lib/xmldsig/reference.rb

```diff
@@ -83,8 +84,8 @@ def digest_method
     end
 
     def digest_value=(digest_value)
-      reference.at_xpath("descendant::ds:DigestValue", NAMESPACES).content =
-          Base64.strict_encode64(digest_value).chomp
+      encoded_value = (@base64_strict_encode == true) ? Base64.strict_encode64(digest_value) : Base64.encode64(digest_value)
```

Comment:
Why `@base64_strict_encode == true` instead of just `@base64_strict_encode`?

```suggestion
      encoded_value = (@base64_strict_encode) ? Base64.strict_encode64(digest_value) : Base64.encode64(digest_value)
```

Reply:
Risk is really low, but I wanted to ensure we only did this on a `boolean` true, not for a `truthy` value.